src/Microsoft.Bcl.HashCode/src/Microsoft.Bcl.HashCode.csproj

```diff
@@ -9,18 +9,17 @@
     <VersionPrefix>1.1.1</VersionPrefix>
     <VersionPrefix Condition="'$(IsPackable)' == 'true'">1.2.0</VersionPrefix>
     <PackageValidationBaselineVersion>1.1.1</PackageValidationBaselineVersion>
+    <IsNetStandard21 Condition="$([MSBuild]::IsTargetFrameworkCompatible('$(TargetFramework)', 'netstandard2.1'))">true</IsNetStandard21>
   </PropertyGroup>
 
-  <PropertyGroup Condition="$([MSBuild]::IsTargetFrameworkCompatible('$(TargetFramework)', 'netstandard2.1'))">
+  <PropertyGroup Condition="'$(IsNetStandard21)' == 'true'">
     <EnableDefaultItems>false</EnableDefaultItems>
   </PropertyGroup>
 
   <ItemGroup>
-    <Compile Include="TypeForwards.cs"
-             Condition="$([MSBuild]::IsTargetFrameworkCompatible('$(TargetFramework)', 'netstandard2.1'))" />
-    <Compile Remove="TypeForwards.cs"
-             Condition="!$([MSBuild]::IsTargetFrameworkCompatible('$(TargetFramework)', 'netstandard2.1'))" />
-    <Compile Include="$(CommonPath)System\LocalAppContextSwitches.Common.cs" Link="Common\System\LocalAppContextSwitches.Common.cs" />
+    <Compile Include="TypeForwards.cs" Condition="'$(IsNetStandard21)' == 'true'" />
+    <Compile Remove="TypeForwards.cs"  Condition="'$(IsNetStandard21)' != 'true'" />
```

Comment:
I think it's better to not introduce a separate `IsNetStandard21` property. We don't do that in runtime in project files and it looks kind of ugly. Also the name of this property is misleading as it is true for both `netstandard2.1` and `NetMinimum`. I would just leave the three conditions as they are and maybe add a separate item group for the `!` not condition so that you put the LocalAppContextSwitches in there to.